pysumma/tests/test_Decisions.py

```diff
@@ -0,0 +1,76 @@
+from pysumma.Decisions import Decisions # This is for testing in cmd window.
+#from ..pysumma.Decisions import Decisions # This is for testing in this python code.
+import unittest
+from shutil import copyfile
+
+class test_decisions_class(unittest.TestCase):
+
+    filename = 'Decision.txt'
+    path = '/home/hydro/Downloads/pysumma_alpha/pysumma/tests/'
```

Comment:
Hard coded paths aren't really portable.  Relative paths for tests usually work better, or using something like `os.path.abspath(__file__)`